cloudmailer/cloudmailer.py

```diff
@@ -720,25 +720,57 @@ def main(argv=None):
 
     if args.schedule and not args.date:
         print ("When scheduling you need to also set a --date")
-        return 22
+        sys.exit(22)
+    # Make sure that the args date is set and have the correct format when scheduling
+    if args.schedule:
+        try:
+            datetime.strptime(args.date, '%Y-%m-%d %H:%M')
+        except ValueError:
+            print ('Starttime %s does not match the format %Y-%m-%d %H:%M')
+            sys.exit(1)
+
+    if args.schedule and args.vms:
+        print ("Cloudmailer does not currently support scheduling downtime on a VM basis. " \
+               "Only hypervisor based downtimes can be scheduled.")
+        sys.exit(1)
+
+    if args.projects and  args.schedule:
+        print ("Cloudmailer does not currently support scheduling downtime on a project basis. " \
+               "Only hypervisor based downtimes can be scheduled.")
+        sys.exit(1)
+
+    return args
+
+def get_template(template_path):
+    """Read template from file"""
+    try:
+        # One should probably define encoding here
+        with open(template_path) as template_file:
+             template = template_file.readlines()
+    except FileNotFoundError:
+        print(f'Template path does not exist: {template_path}')
```

Comment:
To keep syntax similar to Shell scripting (more widely understood), I would use double quotes here. Also because in other strings you used double quotes, so to keep consistency.

Reply:
ok